src/WebJobs.Script/Description/PowerShell/PowerShellFunctionInvoker.cs

```diff
@@ -156,9 +156,12 @@ private static void OutputCollectionDataAdded(object sender, DataAddedEventArgs
         {
             // trace objects written to the output stream
             var source = (PSDataCollection<PSObject>)sender;
-            var msg = source[e.Index].ToString();
-
-            traceWriter.Info(msg);
+            var data = source[e.Index];
```

Comment:
In some PowerShell testing, I was getting null refs here, e.g. if the powershell function writes a null output line